packages/components/src/molecules/Panels/types.ts

```diff
@@ -16,11 +16,13 @@ export type OnStopResize = (position: "left" | "center" | "right", flex: number)
 export type ResizableColumnsPanelType = {
   right: React.ReactNode;
   center?: React.ReactNode;
```

Comment:
Are all these fields are optional?

Reply:
Yes, would you find it useful to add some comments on why is it like this?

Reply:
No thank you, by time I can understand as I learn product more just wanted to double check :)